packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx

```diff
@@ -163,6 +176,12 @@ const Navigation = (props: OwnProps & Props) => {
           </NewCartridge>
         </MenuItem>
       </LinkContainer>
+      <LinkContainer to='/exchange' activeClassName='active'>
+        <MenuItem data-e2e='exchangeLink'>
+          <ExchangeNavItem {...props} />
+        </MenuItem>
+      </LinkContainer>
+      {/* )} */}
```

Comment:
this seems unnecessary...?

Reply:
agreed, must of been an accident and I didn't realize it